
<!DOCTYPE html>
<html>
<head>
  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
  
  <title>eess.IV - 2023-08-19 17:00:00 | Fun Paper</title>
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
  <meta name="description" content="CRC-ICM: Colorectal Cancer Immune Cell Markers Pattern Dataset paper_url: http:&#x2F;&#x2F;arxiv.org&#x2F;abs&#x2F;2308.10033 repo_url: None paper_authors: Zahra Mokhtari, Elham Amjadi, Hamidreza Bolhasani, Zahra Faghih,">
<meta property="og:type" content="article">
<meta property="og:title" content="eess.IV - 2023-08-19 17:00:00">
<meta property="og:url" content="https://nullscc.github.io/2023/08/19/eess.IV_2023_08_19/index.html">
<meta property="og:site_name" content="Fun Paper">
<meta property="og:description" content="CRC-ICM: Colorectal Cancer Immune Cell Markers Pattern Dataset paper_url: http:&#x2F;&#x2F;arxiv.org&#x2F;abs&#x2F;2308.10033 repo_url: None paper_authors: Zahra Mokhtari, Elham Amjadi, Hamidreza Bolhasani, Zahra Faghih,">
<meta property="og:locale" content="en_US">
<meta property="article:published_time" content="2023-08-18T16:00:00.000Z">
<meta property="article:modified_time" content="2023-08-27T10:02:33.614Z">
<meta property="article:author" content="nullscc">
<meta name="twitter:card" content="summary">
  
  
  
<link rel="stylesheet" href="/css/style.css">

  
    <link href="//fonts.useso.com/css?family=Source+Code+Pro" rel="stylesheet" type="text/css">
  
  <!--[if lt IE 9]><script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7/html5shiv.min.js"></script><![endif]-->
  
  

<meta name="generator" content="Hexo 6.3.0"></head>

<body>
<div id="container">
  <div id="wrap">
    <header id="header">
  <div id="banner"></div>
  <div id="header-outer" class="outer">
    <nav id="upper-nav" class="inner">
      <a id="main-nav-toggle" class="nav-icon"></a>
      <div class="sub-nav">
        
        
          <a id="nav-github" class="nav-icon" target="_blank" rel="noopener" href="https://github.com/nullscc"></a>
        
      </div>
    </nav>
    <div id="header-title">
      
        <h1 id="blog-title-wrap">
          <a href="/" id="blog-title">Fun Paper</a>
        </h1>
      
    </div>
    <div id="contenedor">
      <ul class="cube">
        <li class="cara">Paper</li>
        <li class="cara"></li>
        <li class="cara"></li>
        <li class="cara"></li>
        <li class="cara"></li>
        <li class="cara"></li>
      </ul>
    </div>
    <nav id="main-nav">
      
        <a class="main-nav-link" href="/">Home</a>
      
        <a class="main-nav-link" href="/archives">Archives</a>
      
    </nav>
  </div>
</header>

    <div class="outer">
      <section id="main"><article id="post-eess.IV_2023_08_19" class="article article-type-post" itemscope itemprop="blogPost">
  <div class="article-meta">
    <h3 href="/2023/08/19/eess.IV_2023_08_19/" class="article-date">
  <time datetime="2023-08-18T16:00:00.000Z" itemprop="datePublished">2023-08-19</time>
</h3>
    
  <div class="article-category">
    <a class="article-category-link" href="/categories/eess-IV/">eess.IV</a>
  </div>

  </div>
  <div class="article-inner">
  <div class="curve-down">
  <div class="fill-content">
    
    
      <header class="article-header">
        
  
    <h1 class="article-title" itemprop="name">
      eess.IV - 2023-08-19 17:00:00
    </h1>
  

      </header>
    
    <div class="article-entry" itemprop="articleBody">
      
        
        <h2 id="CRC-ICM-Colorectal-Cancer-Immune-Cell-Markers-Pattern-Dataset"><a href="#CRC-ICM-Colorectal-Cancer-Immune-Cell-Markers-Pattern-Dataset" class="headerlink" title="CRC-ICM: Colorectal Cancer Immune Cell Markers Pattern Dataset"></a>CRC-ICM: Colorectal Cancer Immune Cell Markers Pattern Dataset</h2><ul>
<li>paper_url: <a target="_blank" rel="noopener" href="http://arxiv.org/abs/2308.10033">http://arxiv.org/abs/2308.10033</a></li>
<li>repo_url: None</li>
<li>paper_authors: Zahra Mokhtari, Elham Amjadi, Hamidreza Bolhasani, Zahra Faghih, AmirReza Dehghanian, Marzieh Rezaei</li>
<li>For: The paper is focused on the immune checkpoints in the tumor microenvironment of colorectal cancer (CRC) and their potential as therapeutic targets for cancer treatment.* Methods: The paper uses a dataset of 1756 images from 136 patients with CRC, stained with specific antibodies for CD3, CD8, CD45RO, PD-1, LAG3, and Tim3 to investigate the expression of immune checkpoints in the tumor microenvironment.* Results: The paper reports on the differences in immune checkpoint expression between tumors located in the right and left sides of the colon, and the potential implications of these differences for the prognosis of CRC patients.Here is the information in Simplified Chinese text:</li>
<li>for: 这篇论文关注了大肠癌（CRC）的免疫检查点在肿瘤微环境中的表达，以及这些检查点作为癌症治疗的可能性。</li>
<li>methods: 这篇论文使用了136名CRC患者的1756个图像，用具体抗体检测CD3、CD8、CD45RO、PD-1、LAG3和Tim3等免疫检查点的表达。</li>
<li>results: 这篇论文报告了右侧和左侧大肠的肿瘤表达免疫检查点之间的差异，以及这些差异对CRC患者的预后的可能影响。<details>
<summary>Abstract</summary>
Colorectal Cancer (CRC) is the second most common cause of cancer death in the world, ad can be identified by the location of the primary tumor in the large intestine: right and left colon, and rectum. Based on the location, CRC shows differences in chromosomal and molecular characteristics, microbiomes incidence, pathogenesis, and outcome. It has been shown that tumors on left and right sides also have different immune landscape, so the prognosis may be different based on the primary tumor locations. It is widely accepted that immune components of the tumor microenvironment (TME) plays a critical role in tumor development. One of the critical regulatory molecules in the TME is immune checkpoints that as the gatekeepers of immune responses regulate the infiltrated immune cell functions. Inhibitory immune checkpoints such as PD-1, Tim3, and LAG3, as the main mechanism of immune suppression in TME overexpressed and result in further development of the tumor. The images of this dataset have been taken from colon tissues of patients with CRC, stained with specific antibodies for CD3, CD8, CD45RO, PD-1, LAG3 and Tim3. The name of this dataset is CRC-ICM and contains 1756 images related to 136 patients. The initial version of CRC-ICM is published on Elsevier Mendeley dataset portal, and the latest version is accessible via: https://databiox.com
</details>
<details>
<summary>摘要</summary>
便膜癌（CRC）是全球第二常见的癌病原因，可以根据主 tumor 的位置在大肠分为右和左边colon，以及肛门。根据位置，CRC 会出现不同的染色体和分子特征、微生物发生率、生物学发展和结果。已经证明左右两边的肿瘤也有不同的免疫景况，因此预后可能因主 tumor 的位置而异。免疫组件在肿瘤微环境中（TME）plays a critical role in tumor development。一种critical regulatory molecules in TME 是免疫检查点，它们作为免疫 responses 的门槛，调控入侵的免疫细胞功能。压缩性免疫检查点，如PD-1、Tim3和LAG3，是免疫抑制的主要机制，它们在TME 过剩表达，导致肿瘤的进一步发展。这些图像来自于colon这些癌病患者的患者，这些图像用specific抗体CD3、CD8、CD45RO、PD-1、LAG3和Tim3染色。这个 dataset 的名称是CRC-ICM，它包含 1756 个图像，与 136 名病患相关。初版CRC-ICM 已经发表在Elsevier Mendeley 数据集Portail，而最新版本可以通过以下连结获取：https://databiox.com。
</details></li>
</ul>
<hr>
<h2 id="Deformable-Detection-Transformer-for-Microbubble-Localization-in-Ultrasound-Localization-Microscopy"><a href="#Deformable-Detection-Transformer-for-Microbubble-Localization-in-Ultrasound-Localization-Microscopy" class="headerlink" title="Deformable-Detection Transformer for Microbubble Localization in Ultrasound Localization Microscopy"></a>Deformable-Detection Transformer for Microbubble Localization in Ultrasound Localization Microscopy</h2><ul>
<li>paper_url: <a target="_blank" rel="noopener" href="http://arxiv.org/abs/2308.09845">http://arxiv.org/abs/2308.09845</a></li>
<li>repo_url: None</li>
<li>paper_authors: Sepideh K. Gharamaleki, Brandon Helfield, Hassan Rivaz</li>
<li>for: 提高ultrasound imaging中半波长解像力的问题</li>
<li>methods: 使用DEformable DETR（DE-DETR）方法，带有多级弹性注意力分配，从而提高精度和准确性</li>
<li>results: 对比DETR方法，DE-DETR方法在精度和准确性方面均显示了改善，并且在最终超分解图像中也得到了更好的效果<details>
<summary>Abstract</summary>
To overcome the half a wavelength resolution limitations of ultrasound imaging, microbubbles (MBs) have been utilized widely in the field. Conventional MB localization methods are limited whether by exhaustive parameter tuning or considering a fixed Point Spread Function (PSF) for MBs. This questions their adaptability to different imaging settings or depths. As a result, development of methods that don't rely on manually adjusted parameters is crucial. Previously, we used a transformer-based approach i.e. DEtection TRansformer (DETR) (arXiv:2005.12872v3 and arXiv:2209.11859v1) to address the above mentioned issues. However, DETR suffers from long training times and lower precision for smaller objects. In this paper, we propose the application of DEformable DETR (DE-DETR) ( arXiv:2010.04159) for MB localization to mitigate DETR's above mentioned challenges. As opposed to DETR, where attention is casted upon all grid pixels, DE-DETR utilizes a multi-scale deformable attention to distribute attention within a limited budget. To evaluate the proposed strategy, pre-trained DE-DETR was fine-tuned on a subset of the dataset provided by the IEEE IUS Ultra-SR challenge organizers using transfer learning principles and subsequently we tested the network on the rest of the dataset, excluding the highly correlated frames. The results manifest an improvement both in precision and recall and the final super-resolution maps compared to DETR.
</details>
<details>
<summary>摘要</summary>
通过超过半波长的限制， ultrasound 影像中的微ubble (MB) 已经广泛应用于领域中。传统的 MB 位置方法受限于手动调整的参数或者假设固定的点扩散函数 (PSF)  для MBs。这问题了它们在不同的描述设置或深度下的适应性。因此，开发不依赖于手动调整参数的方法是关键。在先前的研究中，我们使用 transformer 基本的方法，即检测转换器 (DETR) （arXiv: 2005.12872v3 和 arXiv: 2209.11859v1）来解决上述问题。然而， DETR 受到训练时间过长和对小对象的精度较低的问题。在这篇论文中，我们提议将 DEformable DETR (DE-DETR) （arXiv: 2010.04159）应用于 MB 位置检测，以减少 DETR 的上述挑战。与 DETR 不同，DE-DETR 使用多尺度可变的注意力来分配注意力，而不是将注意力投射到所有的格子像素上。为评估提议的策略，我们先使用 transfer learning 原则来精心 DE-DETR ，然后在数据集中测试网络，排除高相关性的帧。结果表明，提议的策略在精度和准确性方面得到改进，并且在最终的超解像图中比 DETR 更好。
</details></li>
</ul>
<hr>
<h2 id="Cross-modality-Attention-based-Multimodal-Fusion-for-Non-small-Cell-Lung-Cancer-NSCLC-Patient-Survival-Prediction"><a href="#Cross-modality-Attention-based-Multimodal-Fusion-for-Non-small-Cell-Lung-Cancer-NSCLC-Patient-Survival-Prediction" class="headerlink" title="Cross-modality Attention-based Multimodal Fusion for Non-small Cell Lung Cancer (NSCLC) Patient Survival Prediction"></a>Cross-modality Attention-based Multimodal Fusion for Non-small Cell Lung Cancer (NSCLC) Patient Survival Prediction</h2><ul>
<li>paper_url: <a target="_blank" rel="noopener" href="http://arxiv.org/abs/2308.09831">http://arxiv.org/abs/2308.09831</a></li>
<li>repo_url: None</li>
<li>paper_authors: Ruining Deng, Nazim Shaikh, Gareth Shannon, Yao Nie</li>
<li>for: 预测和诊断结果提高，尤其是诊断非小细胞肺癌（NSCLC）患者的存活情况。</li>
<li>methods: 跨模态注意力基本的多模态融合策略，通过评估不同模式之间的关系来混合特征。</li>
<li>results: 与单模式相比，提出的融合方法在实验中实现了c-指数0.6587，表明可以充分利用不同模式之间的知识，提高诊断和预测的准确性。<details>
<summary>Abstract</summary>
Cancer prognosis and survival outcome predictions are crucial for therapeutic response estimation and for stratifying patients into various treatment groups. Medical domains concerned with cancer prognosis are abundant with multiple modalities, including pathological image data and non-image data such as genomic information. To date, multimodal learning has shown potential to enhance clinical prediction model performance by extracting and aggregating information from different modalities of the same subject. This approach could outperform single modality learning, thus improving computer-aided diagnosis and prognosis in numerous medical applications. In this work, we propose a cross-modality attention-based multimodal fusion pipeline designed to integrate modality-specific knowledge for patient survival prediction in non-small cell lung cancer (NSCLC). Instead of merely concatenating or summing up the features from different modalities, our method gauges the importance of each modality for feature fusion with cross-modality relationship when infusing the multimodal features. Compared with single modality, which achieved c-index of 0.5772 and 0.5885 using solely tissue image data or RNA-seq data, respectively, the proposed fusion approach achieved c-index 0.6587 in our experiment, showcasing the capability of assimilating modality-specific knowledge from varied modalities.
</details>
<details>
<summary>摘要</summary>
肿瘤诊断和存活结果预测是肿瘤治疗的关键因素，可以用于评估治疗效果和将患者分组。医学领域中关于肿瘤诊断的数据非常丰富，包括图像数据和非图像数据，如基因信息。到目前为止，多modal学习已经显示出了提高诊断模型性能的潜力，通过提取和综合不同模式的信息来提高计算机辅助诊断和预测的性能。在这项工作中，我们提出了跨模式关注-基于多模式融合管线，用于汇集不同模式特有的知识来预测患者存活情况。而不是仅将不同模式的特征 concatenate 或 sum，我们的方法会测量不同模式之间的关系，以便在融合多模式特征时进行相应的权重调整。相比单模式，我们的融合方法在实验中实现了c-index 0.6587，超过了使用具体图像数据或 RNA-seq 数据单独进行预测的性能。
</details></li>
</ul>

      
    </div>
    <footer class="article-footer">
      <div class="article-footer-content">
        
        <a data-url="https://nullscc.github.io/2023/08/19/eess.IV_2023_08_19/" data-id="clly3606x00epdd88b1ul2jbw" class="article-share-link">Share</a>
        
      </div>
    </footer>
  </div>
  </div>
  </div>
  
    
<nav id="article-nav">
  
    <a href="/2023/08/19/cs.SD_2023_08_19/" id="article-nav-newer" class="article-nav-link-wrap">
      <strong class="article-nav-caption">Newer</strong>
      <div class="article-nav-title">
        
          cs.SD - 2023-08-19 123:00:00
        
      </div>
    </a>
  
  
    <a href="/2023/08/18/cs.LG_2023_08_18/" id="article-nav-older" class="article-nav-link-wrap">
      <strong class="article-nav-caption">Older</strong>
      <div class="article-nav-title">cs.LG - 2023-08-18 18:00:00</div>
    </a>
  
</nav>

  
</article>


</section>
      
      <aside id="sidebar">
  
    <div class="widget-wrap">
  <h3 class="widget-title">Calendar</h3>
  <div id="calendar"></div>
</div>

  
    
  <div class="widget-wrap">
    <h3 class="widget-title">Categories</h3>
    <div class="widget">
      <ul class="category-list"><li class="category-list-item"><a class="category-list-link" href="/categories/cs-AI/">cs.AI</a><span class="category-list-count">24</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/cs-CL/">cs.CL</a><span class="category-list-count">24</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/cs-CV/">cs.CV</a><span class="category-list-count">24</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/cs-LG/">cs.LG</a><span class="category-list-count">57</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/cs-SD/">cs.SD</a><span class="category-list-count">55</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/eess-AS/">eess.AS</a><span class="category-list-count">29</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/eess-IV/">eess.IV</a><span class="category-list-count">56</span></li></ul>
    </div>
  </div>

  
    
  <div class="widget-wrap">
    <h3 class="widget-title">Archives</h3>
    <div class="widget">
      <ul class="archive-list"><li class="archive-list-item"><a class="archive-list-link" href="/archives/2023/08/">August 2023</a><span class="archive-list-count">104</span></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2023/07/">July 2023</a><span class="archive-list-count">165</span></li></ul>
    </div>
  </div>

  
</aside>
      
    </div>
    <footer id="footer">
  
  <div class="outer">
    <div id="footer-info" class="inner">
      &copy; 2023 nullscc<br>
      Powered by <a href="http://hexo.io/" target="_blank">Hexo</a>
      .
      Theme by <a href="https://github.com/sun11/hexo-theme-paperbox" target="_blank">Paperbox</a>
    </div>
  </div>
</footer>
  </div>
  <nav id="mobile-nav">
  
    <a href="/" class="mobile-nav-link">Home</a>
  
    <a href="/archives" class="mobile-nav-link">Archives</a>
  
</nav>

  

<!-- totop start -->
<div id="totop">
	<a title="To Top"></a>
</div>
<!-- totop end -->

<!-- swiftype search start -->

<!-- swiftype search end -->



<script src="//cdnjs.cloudflare.com/ajax/libs/jquery/2.0.3/jquery.min.js"></script>

<script src="//cdnjs.cloudflare.com/ajax/libs/lrsjng.jquery-qrcode/0.12.0/jquery.qrcode.min.js"></script>


  
<link rel="stylesheet" href="/fancybox/jquery.fancybox.css">

  
<script src="/fancybox/jquery.fancybox.pack.js"></script>



<!-- add calendar widget -->

  <script src="/js/calendar.js"></script>
  <script src="/js/languages.js"></script>
  <script type="text/javascript">
    $(function() {
    
      $('#calendar').aCalendar('en', $.extend('{"months":["January","February","March","April","May","June","July","August","September","October","November","December"],"dayOfWeekShort":["S","M","T","W","T","F","S"],"dayOfWeek":["Sunday","Monday","Tuesday","Wednesday","Thursday","Friday","Saturday"]}', {single:'true', root:'calendar/'}));
    
    });
  </script>



<script src="/js/script.js"></script>


</div>
</body>
</html>
